### ISSUE TITLE
Consistent naming of values camelCase

### ISSUE THREAD

**Issue by clapierre** [w3c/a11y-discov-vocab#7]
under accessibilityFeatures, there are two values currently which do not follow the naming convention.
ChemML
MathML

These I think should be: 

chemML
mathML

**Comment by mattgarrish** [comment]
I believe this is because they are not joined words but the actual names of the technologies.

**Comment by clapierre** [comment]
Agreed.
